_i18n/de/_posts/2021-02-15-white-impressions.md

```diff
@@ -1,10 +1,29 @@
 ---
 layout: post
-title:  "First"
-date:   2021-02-07 17:51:00 +0100
+title:  "Weiße Eindrücke"
+date:   2021-02-15 21:53:00 +0100
 categories: posts
 ---
 
-erster Beitrag.
+Ich lebe in Deutschland seit Jan/2016. 2 Tage nach ich nach Berlin gekommen bin, ich habe zum zweiten Mal in meinem Leben Schnee erlebt (das erster mal war in Turin - Italien). Ich erinnere mich wie glücklich und fasziniert ich war zu weiße Straßen sehen. Wie die Bäume, die vorher hätten keinen Blätter, jetzt wurden von einer dünnen und glatten Schneeschicht bedeckt. In 2016 gab es nicht so viele Schnee: aber es war genüg, um mir glücklich und erfreut zu machen.
 
+Mein 2021 hat (ein bisschen) ähnlich angefangen. Aber dieses mal, mit viel mehr Schnee! Am 2. Januar, ich bin in [Die Rhön](https://de.wikipedia.org/wiki/Rh%C3%B6n) mit meinem Freund und seiner Familie gegangen. Ich hatte keine Ahnung wie der Ort war: ich wusste ich wäre (hoffentlich) etwas Schnee in diesen Tag sehen.
 
+Ich bin an das warme und konstante Wetter aus Fortaleza, meiner Heimatstadt, gewöhnt. Also natürlich alles, was dem Schnee nahe kommt, packt meine Aufmerksamkeit.
+
+Also da war es: Bergen, Schnee und Kinder rutschen die Berghänge hinunter. Die Blicke waren atemberaubend und ich denke, das war die beste Weg, um das Jahr anfangen und die kälte Monate hier wahrhaftig genießen.
+
+An den gleichen Tag bin ich nach Berlin zurückgekommen und war bereit, um das neues Jahr zu beginnen. Die Woche vergingen und alles, was ich durch mein Wohnungsfenster sehe konnte, waren Tagen mit viele wolkige Himmel.
+
+Aber letzte Woche war anders. Die Temperaturen fangen zu sinken an (noch mehr!) und ja, der Schnee kommt! Wir haben auch ein Geschenk bekommen: viel Sonne und blauen Himmel. Meine Mittagspausen waren gefüllt mit Spaziergängen in meinem Kiez , wo ich die Blicken von dem Panke-Kanal und dem Humboldhain Park erlebe könnte.
+
+Nach etwa eine Woche mit negative Temperaturen, einige der nahe gelegenen Seen zu frieren angefangen. Letztes Wochenende bin ich nach [Heiliger See](https://de.wikipedia.org/wiki/Heiliger_See_%28Potsdam%29), in Potsdam,
+gefahren. Es war unglaubich zu sehen, dass den gleichen Seen wo war ich drinnen zu ersten Mal in meine Geburtstag (also im Sommer), ganz gefroren war!
```

Comment:
dass die gleichen Seen, wo ich zum ersten Mal an meinem Geburtstag war

Reply:
How can I specify that I was "inside" the lake? or just by "wo ich zum ersten Mal.." already means that?

Reply:
dann "wo ich zum ersten Mal an meinem Geburtstag drinnen war".

Oder: dass die gleichen Seen, in denen ich zum ersten mal an meinem Geburtstag schwimmen war, [...[